### ISSUE TITLE
Responder to send case details and case observables to a webhook

### ISSUE THREAD

**Issue by xg5-simon** [TheHive-Project/TheHive#1100]
# Responder to send case details and case observables to a webhook

### Request Type
Feature Request

### Work Environment
N/A

### Problem Description
Create a responder that sends case details and case observables to an external webhook.  

### Steps to Reproduce
N/A

### Complementary information
This responder was created to illustrate how to use TheHive4py and CortexUtils.

**Comment by xg5-simon** [comment]
I am currently testing and need to fix two issues before submitting a pull request to the dev branch

- xg5-simon/SendIt#1  
- xg5-simon/SendIt#2

**Comment by nadouani** [comment]
I think this should be submitted to Cortex-Analyzers repo

**Comment by nadouani** [comment]
The thing I'm inconfortable with, is that you provide TheHive creds to an external système that can call TheHive, whereas TheHive should be isolated.

**Comment by xg5-simon** [comment]
I can change it to use the api if more appropriate.

**Comment by nadouani** [comment]
Well technically it's OK, but from a security design perspective, I won't do it that way

**Comment by xg5-simon** [comment]
Any suggestions? I think allowing an analyst to initiate a web hook from the interface has merit.

**Comment by nadouani** [comment]
Yes, but this doesn't need a responder. Are you talking about the WebHooks that you can configure today through the configuration file?
